Service health alerts does not send resource health events

mozlight0302 opened the pull request:
Currently we do not have a feature to send resource health notifications.
(Still in development)

And service health notification documented here does not include resource health events.
So I added a note because current documentation may confuse customers that service health notifications can notify resource health events.

azurecla commented:
Hi __@michielpost__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tamram commented:
Looks good, thanks.

Commit:
Service health alerts does not send resource health events

Currently we do not have a feature to send resource health notifications.
(Still in development)

And service health notification documented here does not include resource health events.
So I added a note because current documentation may confuse customers that service health notifications can notify resource health events.
